types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
:joy: I wasn't arguing the comment to the field, it makes perfect sense.

Just the fact that the go standard doesn't require comment for a field to start with the field name.